west.yml

```diff
@@ -72,7 +72,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: fb6ca20a06ba31a3dc1b86d6986d48ad4a5760b0
```

Comment:
Is this related to the rest of the changes?

Reply:
To setup current limit on bootup I need to put C code within shield. It was impossible in current sdk-zephyr, but in upstream zephyr there is a commit that makes it possible. Cherry-pick from upstream zephyr and sha update is in separate commit. It could be done in a separate PR, but if I did it in a separate PR one could ask in that PR why is this cherry-pick necessary.
So this change is necessary here, but not core of the change.